### PR TITLE
feat: show wordclass/inflectional category on paradigm page

### LINKED ISSUE #550: Show specific wordclass and/or inflectional category on paradigm page
Somewhere on the paradigm page, we should show the same information about the specific wordclass and inflectional category as is already presented on the search/landing page. There's no reason to present less info on the paradigm page. Eventually, we might want to show other info as well, such as the derivational morphological breakdown.

### PR DESCRIPTION
**Pull request by eddieantonio**
Fixes #550

### CODE REVIEW

**Comment by eddieantonio** [comment]
I started to implement this and all the information is TECHNICALLY on the word details page. However, it ugly.

@aarppe + others, please tell me how these look:

word-detail-VII

word-detail-NAD

word-detail-IPC

(I can already tell the duplication of "Particle" will be a problem...)

**Comment by aarppe** [comment]
I had been thinking originally that the user could eventually choose which of the formats for a) the specific word-class (e.g. `VII` vs. emoji) and b) the inflectional category would be shown (e.g. `VII-1n`, or `n-final impersonal inanimate intransitive verb` or `like: kimiwan`), all available in `crk.altlabel.tsv`. Though perhaps on the details page we can show all, as in the example screenshots. And if we want to show everything, I'd provide the plain English `like: X` as well.

In any event, the inflectional category should always tell also the specific wordclass, i.e. `VII` is redundant when `VII-1n` is shown - this should solve the `IPC` problem.

**Comment by eddieantonio** [comment]
> I had been thinking originally that the user could eventually choose which of the formats for a) the specific word-class (e.g. `VII` vs. emoji) and b) the inflectional category would be shown (e.g. `VII-1n`, or `n-final impersonal inanimate intransitive verb` or `like: kimiwan`), all available in `crk.altlabel.tsv`.

Haha, that sounds like _future work_ 😅 

> Though perhaps on the details page we can show all, as in the example screenshots. And if we want to show everything, I'd provide the plain English `like: X` as well.
> 
> In any event, the inflectional category should always tell also the specific wordclass, i.e. `VII` is redundant when `VII-1n` is shown - this should solve the `IPC` problem.

Alrighty, this is what it looks like now (there's some issue with nîpin — not sure why it has the wrong inflectional category on my machine...).

word-detail-IPC
word-detail-NA
word-detail-NDA
word-detail-NDI
word-detail-NI
word-detail-VAI
word-detail-VII
word-detail-VTA
word-detail-VTI

**Comment by atticusha** [comment]
I love this. +1 for some how fitting all the information in one mobile space

File: CreeDictionary/CreeDictionary/views.py [hunk 1]
```diff
@@ -107,7 +111,7 @@ def search_results(request, query_string: str):  # pragma: no cover
     results = Wordform.search_with_affixes(query_string)
     return render(
         request,
-        "CreeDictionary/word-entries.html",
+        "CreeDictionary/search-results.html",
```
Comment (nienna73) [inline CreeDictionary/CreeDictionary/views.py:114]: Are there any tests that use to reference /word-entries that now need to look for /search-results? 
If not, should there be tests for the new /search-results path?

**Review by nienna73 — COMMENTED** [review]
The refactoring and new feature look great, just one comment re: tests.

File: CreeDictionary/CreeDictionary/views.py (hunk 1 above)
Comment (eddieantonio) [reply CreeDictionary/CreeDictionary/views.py:114]: This a rename (refactor) to a _template file_. It's not a URL path! The URLs have all stayed the same. That said, I'd be interested in hearing how we can strengthen our tests!

Comment (eddieantonio) [reply CreeDictionary/CreeDictionary/views.py:114]: Here is the renamed file in question: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/616/files#diff-58ca8956f4bfbe13e67b376ee3b4e5d1ab1445330bcda3eb43c2125bd57b1f80

Comment (nienna73) [reply CreeDictionary/CreeDictionary/views.py:114]: Ah I see. We should go over the tests in a different PR if they aren't directly related to these changes, then.

**Review by nienna73 — APPROVED** [review]

**Comment by aarppe** [comment]
The current formattings look nice. As a further detail, I'd think we'd want to provide the stem as well, though I realize that will add to the clutter. In addition, eventually we'd want to show the derivational breakdown, and the semantic classification according to CW and/or RW.

**Pull request merged**